hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/StorageVolumeChecker.java

```diff
@@ -109,11 +120,6 @@ public HddsVolumeChecker(ConfigurationSource conf, Timer timer)
     this.timer = timer;
 
     DatanodeConfiguration dnConf = conf.getObject(DatanodeConfiguration.class);
-    /**
-     * Maximum number of volume failures that can be tolerated without
-     * declaring a fatal error.
-     */
-    int maxVolumeFailuresTolerated = dnConf.getFailedVolumesTolerated();
 
     minDiskCheckGapMs = conf.getTimeDuration(
         DFSConfigKeysLegacy.DFS_DATANODE_DISK_CHECK_MIN_GAP_KEY,
```

Comment:
I think, its better to avoid any legacy configs in the code. can be done later as well.

Reply:
Ah yes, there are many legacy configs left, 2-3 are disk checker related, will fix it here.